interfaces/builtin/core_support.go

```diff
@@ -24,17 +24,32 @@ import (
 )
 
 const coreSupportConnectedPlugAppArmor = `
-# Description: Can control all aspects of systemd via the systemctl command
-# and update rsyslog configuration. The interface allows execution of the
-# systemctl binary unconfined. As such, this gives device ownership to the
-# snap.
+# Description: Can control all aspects of systemd via the systemctl command,
+# update rsyslog configuration, update systemd-timesyncd configuration and
+# update/apply sysctl configuration. The interface allows execution of the
+# systemctl binary unconfined and modifying all sysctl configuration. As such,
+# this gives device ownership to the snap.
 
 /bin/systemctl Uxr,
 
 # Allow modifying rsyslog configuration for such things as remote logging. For
 # now, only allow modifying NN-snap*.conf and snap*.conf files.
 /etc/rsyslog.d/{,*}                     r,
 /etc/rsyslog.d/{,[0-9][0-9]-}snap*.conf w,
+
+# Allow modifying /etc/systemd/timesyncd.conf for adjusting systemd-timesyncd's
+# timeservers
+/etc/systemd/timesyncd.conf rw,
+
+# Allow modifying sysctl configuration and applying the changes. For now, allow
+# reading all sysctl files but only allow modifying NN-snap*.conf and
+# snap*.conf files in /etc/sysctl.d.
+/etc/sysctl.conf                       r,
+/etc/sysctl.d/{,*}                     r,
+/etc/sysctl.d/{,[0-9][0-9]-}snap*.conf w,
+/{,usr/}{,s}bin/sysctl                 ixr,
```

Comment:
Yes, I did, which is why I added the @{PROC}/sys after. systemctl is Uxr for unfortunate reasons, but I didn't see any reason to not confine sysctl.